### ISSUE TITLE
[Selector module] Add the method in ` MultipleComparisons`

### ISSUE THREAD

**Issue by FanwangM** [theochem/Selector#53]
Ramon's group has a clever way of doing diverse selector and we have an in-house implementation and should merge it to this repo.

**Comment by PaulWAyers** [comment]
Reference: https://github.com/ramirandaq/MultipleComparisons

**Comment by PaulWAyers** [comment]
I'm not exactly sure what algorithm is used for diverse-selection here, but this quantifies diversity of the subset well, and also could obviously be combined with a greedy method like used in determinantal point processes.

**Comment by ramirandaq** [comment]
Hi, just a quick recap of the diversity pickers that we have implemented:

1- Max_nDis (https://jcheminf.biomedcentral.com/articles/10.1186/s13321-021-00504-4): starts from a random point and looks to minimize the extended similarity of the selected objects.

2- ECS-MeDiv (https://chemrxiv.org/engage/chemrxiv/article-details/62449b5b3b5f991e7cca0670): this is a bit more nuanced because we also implemented several possible starting points: random, medoid (most representative element of the set), outlier (we perform the medoid and outlier selection in O(N)). Then we also seek to minimize the extended similarity, but we added an extra step: in case there are several elements that give the same extended similarity, we select the one that also minimizes the binary similarity.
